src/mmap_unix.rs

```diff
@@ -160,6 +176,47 @@ impl MmapRegion {
             file_offset,
             prot,
             flags,
+            owned: true,
+        })
+    }
+
+    /// Creates a MmapRegion instance for an externally managed mapping.
```

Comment:
I think it would be useful for other people that look at the interface of MmapRegion to also to give an example of situation in which people can choose to use this function as opposed to the other builders. This helps people that look at the vm-memory interface for the first time to decide when to use one over the other.

Reply:
Sounds like a good idea to me. I'm going to add it.

Reply:
Thanks!